packages/odyssey-react-mui/src/SearchField.tsx

```diff
@@ -100,6 +106,7 @@ const SearchField = forwardRef<HTMLInputElement, SearchFieldProps>(
       testId,
       translate,
       value,
+      variant = "outline",
```

Comment:
I'd like consumers to explicitly set this property, but since this is a new addition and since we don't wanna break existing users, I'm fine with this default.

I'm wondering if it's possible to `@deprecate` not having a variant? Can you check if that's possible in TypeScript?

My goal is to make it yellow-underline folks when not passing the variant explicitly, but let it not break if they don't pass one.

Reply:
I haven't been able to find a clean way of doing this yet. One thing to consider is that the new variant really only makes sense once we have the gray background; we're just setting it up for that future. So right now it really should be `outline` 99% of the time.